mcs/class/corlib/Test/System.Reflection.Emit/SaveTest.cs

```diff
@@ -67,9 +67,31 @@ public class SaveTest
 		0x79, 0xC0, 0x9B, 0x5F, 0x34, 0x86, 0xB2, 0xDE, 0xC4, 0x19, 0x84, 0x5F, 
 		0x0E, 0xED, 0x9B, 0xB8, 0xD3, 0x17, 0xDA, 0x78 };
 
+	string tempDir = Path.Combine (Path.GetTempPath (), typeof (SaveTest).FullName);
+
+	[SetUp]
+	protected void SetUp ()
```

Comment:
I think FixtureSetUp should be enough

Reply:
I copied it from TypeBuilderTest where the same assembly name is used in all the tests so it makes sense to create/delete the dir after each test. Given SaveTest has just one test right now it doesn't really matter so unless you feel strongly I'd keep it as it is.